src/typespec.ts

```diff
@@ -17,6 +17,7 @@ export interface TypeSpec {
   readonly properties: ReadonlyArray<TypeSpec> | undefined;
   readonly hasAdditionalProperties: boolean;
   readonly additionalPropertiesType: TypeSpec | undefined;
+  readonly "x-qualified-classname"?: string;
```

Comment:
Could we / should we make this output type more friendly for accessing? It's going to lead to cleaner code if we can use `xQualifiedClassname` or `crl8QualifiedClassname` or something